sdk/core/core-rest-pipeline/src/util/file.ts

```diff
@@ -1,8 +1,11 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { isNodeLike } from "@azure/core-util";
-import { isNodeReadableStream } from "./typeGuards.js";
+import {
```

Comment:
nit: seems this is only for internal. Maybe we can get rid of this file and use these from unbranded directly?

Reply:
thanks, good catch. Since we don't actually need createFile and friends in unbranded (the codegen doesn't use that) I removed it in unbranded instead and added some wrapping logic to the core-rest-pipeline multipartPolicy to pull the actual content out of any fake `File`s created using createFile.